src/main/java/core/basesyntax/file/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.file;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String fileName) {
+        try {
+            Files.write(Paths.get(fileName), data.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file" + fileName, e);
+        }
+    }
```

Comment:
Remember to follow the encapsulation principle. The method 'write' should not be public if it's not intended to be exposed as part of the API. If it's only used within the package, package-private would be more appropriate.